### DISCUSSION TITLE
Script title pattern causing SQL tab to show names redundantly

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#13370] [Q&A]
These are my settings:

![image](https://user-images.githubusercontent.com/74785256/127508480-bde9beda-86ff-4088-a806-f230ebbd3d35.png)

However, this is my file tab:

![image](https://user-images.githubusercontent.com/74785256/127508530-f31e42ab-f04d-4279-a112-3457fc17e14a.png)

Notice that it shows 4 things in there, not 3 as in settings: connection database [???] filename

Here's what my connection looks like:

![image](https://user-images.githubusercontent.com/74785256/127508794-3dafaaa4-308b-4925-b20f-5b52069f9286.png)

So, "connection" seems correct, and "database" seems correct, but either that or [???] makes no sense to be there, and seems redundant and so expanding the tab length unnecessarily.

What is this extra name in there referring to? Is it redundant or referring to something not specified in settings? I seem to be able to have no effect on it.

**Comment by LonwoLonwo** [comment]
Hi @givihuda 
Name of MySQL Database includes datasource name in square brackets, yes.  
PostgreSQL or SQL Server - don't.
So it happened historically.

**Reply by givihuda** [reply]
So datasource name is ${database}?

So ${database} for MySQL literally prints in the Script title pattern (the tab name):

 `laravel [laravel]`

whereas PostgreSQL would print instead:

`laravel`

Am I correct in understanding what you've said?

**Reply by LonwoLonwo** [reply]
Yes, you are totally right.
